qabelbox/src/main/java/de/qabel/qabelbox/services/QabelService.java

```diff
@@ -244,45 +136,19 @@ public boolean onUnbind(Intent intent) {
      */
     private void initServiceResources() {
```

Comment:
Is this javadoc comment "Requires global ResourceActor to be ready." still valid? (Comment not printed in this diff)